src/main/java/io/demo/extractor/FieldExtractor.java

```diff
@@ -25,7 +25,8 @@
 	@Override
 	public List<String> getSupportedTypes() {
 		return Lists.newArrayList(
-				"Form"
+				"Form",
+				"Header"
```

Comment:
Does not seem right that we have to add built-in widget types here, worth checking